wsmaster/codenvy-hosted-api-organization/src/main/java/com/codenvy/organization/api/resource/SuborganizationResourcesProvider.java

```diff
@@ -83,7 +83,7 @@ public SuborganizationResourcesProvider(AccountManager accountManager,
         }
 
         // given account is suborganization's account and can have resources provided by parent
-        List<? extends Resource> parentResources = usageManagerProvider.get().getAvailableResources(parent);
+        List<? extends Resource> parentResources = usageManagerProvider.get().getTotalResources(parent);
```

Comment:
i would call the variable differently e.g. `parentTotal` or `parentAvaialble` then when you use it in the code it would be more _declarative_, so you could determine the problem earlier